horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/UserServiceImpl.java

```diff
@@ -58,29 +55,41 @@ public class UserServiceImpl implements UserService {
         return backend.get().getTeams();
     }
 
-    @WithRoles(addUsername = true)
+    @Transactional
+    @WithRoles(extras = Roles.HORREUM_SYSTEM)
     @Override public String defaultTeam() {
         UserInfo userInfo = UserInfo.findById(identity.getPrincipal().getName());
         return userInfo != null ? userInfo.defaultTeam : null;
```

Comment:
Is it "ok" to return null here?

Reply:
probably safer to return `""` when `userInfo` exists.